homeassistant/components/sensor/homematicip_cloud.py

```diff
@@ -103,23 +83,18 @@ def available(self):
     def device_state_attributes(self):
         """Return the state attributes of the access point."""
         return {
-            ATTR_HOME_LABEL: self._home.label,
-            ATTR_HOME_ID: self._home.id,
+            ATTR_HOME_NAME: self._home.name,
+            ATTR_CONNECTED: self._home.connected,
```

Comment:
We should not add this. It's already exposed as available attribute. So when a sensor state is not unavailable, you know it's available.